package/gluon-core/luasrc/lib/gluon/upgrade/140-firewall-rules

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/lua
-local site = require 'gluon.site_config'
```

Comment:
Fold this into the first patch, which adds this line.

Reply:
done.

Reply:
done